tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/PackageExplorer.java

```diff
@@ -57,6 +57,7 @@ private void openProject(SWTBotTreeItem project) {
     }
 
     private SWTBotTreeItem getProject(String projectName) {
+        bot.viewByTitle(PACKAGE_EXPLORER).show();
```

Comment:
why is this required?
I mean, I don't say it's wrong, but if that's required maybe you should put a comment on that so we're sure we won't remove that in the feature when we clean up tests from Cucumber

Reply:
This is required otherwise the test will not start, if something different happens before. Example: Open a view and do something there.

Reply:
Isn't this already part of the master branch after your PR on test restructuring? Am I missing something?

Reply:
You are correct.

Reply:
No, it is not in the master branch.